main.tf

```diff
@@ -220,113 +229,140 @@ resource "aws_security_group" "instances" {
   from_port   = 6443
   to_port     = 6443
   protocol    = "tcp"
 
-  security_group_id = "${aws_security_group.provisioner.id}"
-  cidr_blocks       = ["${var.provisioner_cidr_block}"]
+  security_group_id = aws_security_group.provisioner.id
+  cidr_blocks       = [var.provisioner_cidr_block]
 }
 
 resource "aws_security_group_rule" "provisioner_secgrp_ingress_6443" {
-  count = "${var.use_provisioner_secgrp ? 1 : 0}"
+  count = var.use_provisioner_secgrp ? 1 : 0
 
   type        = "ingress"
   description = "RKE K8s endpoint verification"
   from_port   = 6443
   to_port     = 6443
   protocol    = "tcp"
 
-  security_group_id        = "${aws_security_group.provisioner.id}"
-  source_security_group_id = "${var.provisioner_security_group}"
+  security_group_id        = aws_security_group.provisioner.id
+  source_security_group_id = var.provisioner_security_group
 }
 
 resource "aws_security_group_rule" "provisioner_cidr_ingress_443" {
-  count = "${var.use_provisioner_secgrp ? 0 : 1}"
+  count = var.use_provisioner_secgrp ? 0 : 1
 
   type        = "ingress"
   description = "Ranchhand cluster verification"
   from_port   = 443
   to_port     = 443
   protocol    = "tcp"
 
-  security_group_id = "${aws_security_group.provisioner.id}"
-  cidr_blocks       = ["${var.provisioner_cidr_block}"]
+  security_group_id = aws_security_group.provisioner.id
+  cidr_blocks       = [var.provisioner_cidr_block]
 }
 
 resource "aws_security_group_rule" "provisioner_secgrp_ingress_443" {
-  count = "${var.use_provisioner_secgrp ? 1 : 0}"
+  count = var.use_provisioner_secgrp ? 1 : 0
 
   type        = "ingress"
   description = "Ranchhand cluster verification"
   from_port   = 443
   to_port     = 443
   protocol    = "tcp"
 
-  security_group_id        = "${aws_security_group.provisioner.id}"
-  source_security_group_id = "${var.provisioner_security_group}"
+  security_group_id        = aws_security_group.provisioner.id
+  source_security_group_id = var.provisioner_security_group
 }
 
 #------------------------------------------------------------------------------
 # Provisioner
 #------------------------------------------------------------------------------
 module "ranchhand" {
-  source = "github.com/dominodatalab/ranchhand/terraform"
-
-  node_ips = ["${split(",", replace(join(",", formatlist("%s:%s", aws_instance.this.*.public_ip, aws_instance.this.*.private_ip)), "/^:|(,):/", "$1"))}"]
-
-  distro           = "${var.ranchhand_distro}"
-  release          = "${var.ranchhand_release}"
-  working_dir      = "${var.ranchhand_working_dir}"
-  cert_dnsnames    = ["${concat(list(aws_elb.this.dns_name), var.cert_dnsnames)}"]
-  cert_ipaddresses = ["${var.cert_ipaddresses}"]
-
-  ssh_username   = "${var.ssh_username}"
-  ssh_key_path   = "${var.ssh_key_path}"
-  ssh_proxy_user = "${var.ssh_proxy_user}"
-  ssh_proxy_host = "${var.ssh_proxy_host}"
-
-  admin_password = "${var.admin_password}"
+  source = "github.com/dominodatalab/ranchhand?ref=DOM-14267/terraform"
```

Comment:
I will make a new ranchhand tag and hardcode it to that tag once the companion ranchhand PR (https://github.com/dominodatalab/ranchhand/pull/31) gets merged (and before I merge this).